boot/boot.js

```diff
@@ -1444,6 +1444,7 @@ $tw.loadTiddlersBrowser = function() {
 	for(var t=0; t<containerIds.length; t++) {
 		$tw.wiki.addTiddlers($tw.wiki.deserializeTiddlers("(DOM)",document.getElementById(containerIds[t])));
 	}
+	callback();
```

Comment:
`loadTiddlersNode` is also synchronous, so it would apply the same way. I had to do both synchronous or neither, it seemed, but I technically would not have to. I could just call `finishStartup` after the call to `loadTiddlersBrowser`, however this would allow someone to overload the `loadTiddlersBrowser` as well if they wanted to. And there are reasons you might want to do that, so I would rather just leave a callback on both.